lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+import pymongo
+from pymongo import MongoClient
+from pymongo.errors import PyMongoError
+
+class Command(BaseCommand):
+    args = '<old_username> <new_username>'
+    help = 'Modify the username of an existing user'
+
+    def handle(self, *args, **options):
+        if len(args) != 2:
+            raise CommandError('Usage is rename_user {0}'.format(self.args))
+
```

Comment:
Can we factor the operational part of this code out from the interface to django management commands?  B/C ultimately we'll want to surface this in the /sysadmin page